site/_docs/continuous-integration.md

```diff
@@ -38,7 +38,7 @@ Save the commands you want to run and succeed in a file: `./script/cibuild`
 
 ### The HTML Proofer Executable
 
-{% highlight bash %}
```

Comment:
I always wondered this, does Rogue and Pygments really treat Bash and Sh differently?  I mean one might have more tokens I guess but I had always figured the former would just inherit the latter.